t/e2e/whisper/mailservice_test.go

```diff
@@ -6,9 +6,9 @@ import (
 	"net/http"
 	"testing"
 
-	"github.com/status-im/status-go/e2e"
 	"github.com/status-im/status-go/geth/node"
 	"github.com/status-im/status-go/geth/params"
+	e2e "github.com/status-im/status-go/t/e2e"
```

Comment:
wouldn't it be exported as `e2e` anyway?

Reply:
I thought so, but goimports keeps saving it like this and simple changing it and ignore goimports results in a failed test. Does it mean something else is wrong about the package structure, or can it stay like this?

Reply:
hmm, I don't see anything wrong with the package structure there...
ok, we can keep it as is then

Reply:
Great, let me know if I should change anything else then. :)